build-system/tasks/helpers.js

```diff
@@ -20,13 +20,18 @@ const crypto = require('crypto');
 const debounce = require('debounce');
 const del = require('del');
 const esbuild = require('esbuild');
+/** @type {Object} */
```

Comment:
Why is this annotation and the one below it needed?

Reply:
In this case esbuild expects define to be of type { [key: string]: string } while this is { [key: string]: boolean }.
I'm not sure if this is actually an issue but at least for now I would like to avoid changing the functionality. I do plan to reach out to @dvoytenko about this though.

The second instance below is an issue with TypeScript detecting the default export of the module as not being callable.

Reply:
I seee okay thanks.